pygen/pygen_src/isa/riscv_instr.py

```diff
@@ -12,7 +12,6 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 Regression script for RISC-V random instruction generator
```

Comment:
This line seems to be copied to all license headers, please remove them

Reply:
Sure, I will.change accordingly